src/core/PerspectivismCore.ts

```diff
@@ -167,18 +163,47 @@ export default class PerspectivismCore {
         if (neighbourHoodExp == null) {
             throw Error(`Could not find neighbourhood with URL ${url}`);
         };
-        console.log("Core.installNeighbourhood: Got neighbourhood", neighbourHoodExp);
+        console.log("Core.installNeighbourhood(): Got neighbourhood", neighbourHoodExp);
         let neighbourhood: Neighbourhood = neighbourHoodExp.data;
         this.languageController.installLanguage(neighbourhood.linkLanguage, null);
         
         return this.#perspectivesController!.add("", url, neighbourhood);        
     }
 
-    async languageCloneHolochainTemplate(languagePath: string, dnaNick: string, uid: string): Promise<LanguageRef> {
-        if (!this.#languageFactory) {
-            throw Error("Language factory was not started when calling core.initLanguages()")
+    async languageApplyTemplateAndPublish(sourceLanguageHash: string, templateData: object): Promise<LanguageRef> {
+        if (!this.#languageController) {
+            throw Error("LanguageController not been init'd. Please init before calling language altering functions.")
+        };
+        let languageObject = await this.#languageController.languageApplyTemplateOnSource(sourceLanguageHash, templateData);
+        try {
```

Comment:
Here we could just call the function below, right?

Reply:
Oh, now I've spotted the difference. Was surprised to find that `languagePublish` also applies template params, but that's cool.

At least we can extract the `try` block, which is the same, right?